File: 2nd year/1st semester/MAP/CS_ToyLanguageInterpreter/MAP_Interpreter/View/ExitCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MAP_Interpreter.View
{
    public class ExitCommand : Command
    {
        public ExitCommand(string key, string description) : base(key, description)
        {
            
        }

        public override void execute()
        {
            System.Environment.Exit(1);
        }
    }
}

File: 3rd year/2nd semester/DesignPatterns/DesignPatternsProject/src/com/company/view/commands/ExitCommand.java
package com.company.view.commands;

public class ExitCommand extends Command {
    public ExitCommand(String key, String description) {
        super(key, description);
    }

    @Override
    public void execute() {
        System.out.println("Goodbye");
        System.exit(0);
    }
}
